tests/testthat/test-cube-nway.R

```diff
@@ -70,12 +70,13 @@ with_test_authentication({
                     3, 4,
                     3, 6,
                     4, 3,
-                    5, 1
+                    5, 1,
+                    0, 0
                 ),
-                dim = c(2L, 5L),
+                dim = c(2L, 6L),
```

Comment:
Won't this still be `5L` in the current world?

Reply:
No; for this test I decided to explicitly add the "No Data" category to the fixtutre (see line 5).